packages/config/src/defaultTestConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { PrismaConfig } from './PrismaConfig'
+
+/**
+ * This default config can be used as basis for unit and integration tests.
+ */
+export function defaultTestConfig(): PrismaConfig {
+  return {
+    experimental: true,
+    loadedFromFile: null,
+  }
+}
```

Comment:
What does this accomplish that `packages/config/src/defaultConfig.ts` doesn't already?

Reply:
For the moment they are the same, indeed. I wanted to create a clear distinction from the start between a default config suitable for tests, and the default config for production use.
In my experience you'll likely have different config (e.g. paths, logging flags, ...) during tests eventually and then having to untangle which defaultConfig call is used in tests and which in actual code gets tedious.